combined_browser_metrics/views/active_users_aggregates.view.lkml

```diff
@@ -109,9 +109,162 @@ dimension: ytd_only {
   sql:  (EXTRACT(DAYOFYEAR FROM ${submission_date}) < EXTRACT(DAYOFYEAR FROM CURRENT_DATE()));;
 }
 
+  # Hide metric columns showing as dimensions
+  dimension: mau {
+    hidden: yes
+    sql: ${TABLE}.mau ;;
+  }
+
+  dimension: dau {
+    hidden: yes
+    sql: ${TABLE}.dau ;;
+  }
+
+  dimension: wau {
+    hidden: yes
+    sql: ${TABLE}.wau ;;
+  }
+
+  dimension: new_profiles {
+    hidden: yes
+    sql: ${TABLE}.new_profiles ;;
+  }
+
+  dimension: ad_clicks {
+    hidden: yes
+    sql: ${TABLE}.ad_clicks ;;
+  }
+
+  dimension: organic_search_count {
+    hidden: yes
+    sql: ${TABLE}.organic_search_count ;;
+  }
+
+  dimension: search_count {
+    hidden: yes
+    sql: ${TABLE}.search_count ;;
+  }
+
+  dimension: search_with_ads {
+    hidden: yes
+    sql: ${TABLE}.search_with_ads ;;
+  }
+
+  dimension: uri_count {
+    hidden: yes
+    sql: ${TABLE}.uri_count ;;
+  }
+
+  dimension: active_hours {
+    hidden: yes
+    sql: ${TABLE}.active_hours ;;
+  }
+
+  # Define measures
   measure: daily_active_users {
     label: "DAU"
     type:  sum
     sql: (${TABLE}.dau) ;;
   }
+
+  measure: weekly_active_users {
+    label: "WAU"
+    type: sum
+    sql:  ${TABLE}.wau ;;
+  }
+
+  measure: monthly_active_users {
+    label: "MAU"
+    type: sum
+    sql:  ${TABLE}.mau ;;
+  }
+
+  measure: new_profile {
+    label: "New Profile"
```

Comment:
Looker's convention is (1) replace `_` with space and (2) capitalize, so some of these like this one will automatically turn into "New Profile" but I think it is better to be explicit 👍